### PR TITLE
Fix puppeteer viewport issues, now only tweet ID can be used. Eliminate -u flag

### LINKED ISSUE #645: CLI puppetter should work with only -i option?
`node . -u https://twitter.com/naval/status/1002103360646823936 -p`

Not priority. But good if only need to pass ID

### PR DESCRIPTION
**Pull request by PuruVJ**
## Description 
- Fix puppeteer viewport - Can be changed using a single `factor` variable. See `twitter-puppeteer/index.js` Line 12
- No need to pass URl with `-u` now. 

```bash
node . -i 1324263512621883393 -o simon -p
```

works perfectly.

## Screenshots

Zoomed out
image

PDF itself(Needs more work, includes non-thread replies): 
[naval.pdf](https://github.com/twindle-co/twindle/files/5499486/naval.pdf)

Closes #645 

More stuff showing and filtering depends on @vijaya-bhaskar now.

### CODE REVIEW

**Comment by PuruVJ** [comment]
Don't close. There's a bug.

**Comment by PuruVJ** [comment]
Now it can be closed

File: twindle-cli/twitter-puppeteer/index.js [hunk 1]
```diff
@@ -1,19 +1,59 @@
...
-  return { data: tweets };
+const puppeteer = require("puppeteer");
+const { waitFor } = require("../utils/helpers");
+
+/**
+ * Get tweets(even older than 7 days) using puppeteer
+ * @param {string} tweetID
+ */
+const getTweet = async (tweetID) => {
+  const pageURL = `https://twitter.com/anyone/status/${tweetID}`;
```
Comment (johnjacobkenny) [inline twindle-cli/twitter-puppeteer/index.js:9]: Does this enable accessing any tweet without knowing the user?

File: twindle-cli/twitter-puppeteer/index.js [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+
+  try {
+    // Modify this variable to control the size of viewport
+    const factor = 0.1;
+    const height = Math.floor(2000 / factor);
+    const width = Math.floor(1700 / factor);
+
+    const browser = await puppeteer.launch({
+      headless: false,
```
Comment (johnjacobkenny) [inline twindle-cli/twitter-puppeteer/index.js:18]: Can be set back to true?

File: twindle-cli/twitter-puppeteer/index.js [hunk 3]
```diff
@@ -1,19 +1,59 @@
...
+      args: [`--force-device-scale-factor=${factor}`, `--window-size=${width},${height}`],
+    });
+
+    const page = await browser.newPage();
+
+    await page.goto(pageURL, {
+      waitUntil: "networkidle2",
+    });
+
+    await waitFor(4000);
```
Comment (johnjacobkenny) [inline twindle-cli/twitter-puppeteer/index.js:32]: Not sure, but there is some builtin to puppeteer that does the same? @vijaya-bhaskar

**Review by johnjacobkenny — APPROVED** [review]
Looks good I think.. just see if any changes need to be made as per comments

File: twindle-cli/twitter-puppeteer/index.js (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/twitter-puppeteer/index.js:9]: Yup. Twitter will automatically redirect to the right person. You could replace anyone with realDonaldTrump 🤣🤣🤣

(DON'T, BUT)

File: twindle-cli/twitter-puppeteer/index.js (hunk 2 above)
Comment (PuruVJ) [reply twindle-cli/twitter-puppeteer/index.js:18]: Definitely, but false for testing purposes

File: twindle-cli/twitter-puppeteer/index.js (hunk 3 above)
Comment (PuruVJ) [reply twindle-cli/twitter-puppeteer/index.js:32]: Tried lot of in built solution, but nothing worked. It simply wouldn't wait for rest of the tweets to load.

But wait, I can try one more thing.

Comment (PuruVJ) [reply twindle-cli/twitter-puppeteer/index.js:32]: I'll try that fix later again extensively

**Comment by PuruVJ** [comment]
Closed. #653 combines this one

**Pull request closed without merging**